suite-native/message-system/src/components/ContextMessage.tsx

```diff
@@ -0,0 +1,170 @@
+        backgroundColor: 'backgroundAlertBlueSubtleOnElevation0',
+        icon: 'info',
+        iconColor: 'iconAlertBlue',
+        iconBackgroundColor: 'backgroundAlertBlueSubtleOnElevation1',
+    },
+    warning: {
+        backgroundColor: 'backgroundAlertYellowSubtleOnElevation0',
+        icon: 'warningTriangle',
+        iconColor: 'iconAlertYellow',
+        iconBackgroundColor: 'backgroundAlertYellowSubtleOnElevation1',
+    },
+    critical: {
+        backgroundColor: 'backgroundAlertRedSubtleOnElevation0',
+        icon: 'warningOctagon',
+        iconColor: 'iconAlertRed',
+        iconBackgroundColor: 'backgroundAlertRedSubtleOnElevation1',
+    },
+} as const satisfies Record<Variant, ContextMessageStyle>;
+
+const messageContainerStyle = prepareNativeStyle<{ backgroundColor: Color }>(
+    (utils, { backgroundColor }) => ({
+        backgroundColor: utils.colors[backgroundColor],
+        paddingVertical: utils.spacings.medium,
+        paddingLeft: utils.spacings.small / 2,
+        paddingRight: utils.spacings.medium,
+        borderRadius: utils.borders.radii.medium,
+    }),
+);
+
+const messageContentStyle = prepareNativeStyle(() => ({
+    flexDirection: 'row',
+    flexShrink: 1,
+}));
+
+const IconContainerStyle = prepareNativeStyle(utils => ({
+    padding: utils.spacings.small * 1.5,
+    borderRadius: utils.borders.radii.round,
+    justifyContent: 'center',
+    alignItems: 'center',
+}));
+
+const messageTextContainerStyle = prepareNativeStyle(() => ({
+    flexShrink: 1,
+    alignItems: 'flex-start',
+}));
+
+const MessageCloseButton = ({
+    backgroundColor,
+    onClose,
+}: {
+    backgroundColor: Color;
+    onClose: () => void;
+}) => (
+    <TouchableOpacity onPress={onClose}>
+        <RoundedIcon
+            name="close"
+            iconSize="medium"
+            containerSize={44}
+            backgroundColor={backgroundColor}
+        />
+    </TouchableOpacity>
+);
+
+export const ContextMessage = ({ messageId }: ContextMessageProps) => {
+    const dispatch = useDispatch();
+    const { applyStyle } = useNativeStyles();
+
+    const message = useSelector((state: MessageSystemRootState) =>
+        selectMessageById(state, messageId),
+    );
+
+    if (G.isNull(message)) return null;
+
+    // We use only English locale in suite-native so far. When the localization to other
+    // languages is implemented, the language selection logic has to be added here.
+    const messageContent = message.content.en;
+
+    const messageLinkLabel = message.cta?.label.en;
+    const messageLink = message.cta?.link;
+    const isLinkDisplayed = messageLinkLabel && messageLink;
+
+    const isMessageDismissible = message.dismissible;
+
+    const handleDismissMessage = () => {
+        // So far we do not distinguish between categories in @suite-native/message-system.
+        // For this reason, the message has to be dismissed for all the categories.
+        allMessageCategories.forEach(category => {
```

Comment:
It looks to me that you implemented only "banner" category, which is totally fine and you can use just that one category. No need to mention the others or dismiss the message from it.